lib/shoulda/matchers/action_controller/session_store.rb

```diff
@@ -20,7 +20,7 @@ def has_value?(expected_value)
         end
 
         def empty?
-          session.empty?
+          session&.empty?
```

Comment:
Per the argument I was making before, I don't think we don't need these `&.` checks, would you mind removing these?

Reply:
Sure :sparkles: